etest/docker/container.py

```diff
@@ -16,8 +17,7 @@
 
 def commit(container: Container, tag: str, repository: str, *args, **kwargs):
     """Commit a Docker container."""
-    print(repository)
-    repo = repository.replace("=", "").replace("[", "").replace("]", "").replace(",", "")
+    repo = re.sub("[\\[\\]=,]", "", repository).lower()
```

Comment:
I'd much rather avoid a regex as it's needlessly complex for something so simple.

repository = str([c for c in repository if c not in "=[],"]).lower()

Reply:
thats actually a very good idea

Reply:
should I move this to a new branch?

Reply:
Branches aren't committed so not necessary but you should update the commit message appropriately.

Reply:
I'll look into it.